lib/readers/Link.js

```diff
@@ -5,7 +5,20 @@ import logger from "@ui5/logger";
 const log = logger.getLogger("resources:readers:Link");
 
 /**
- * A reader that allows removing paths segments from all resources passed through it.
+ * A reader that allows for rewriting paths segments of all resources passed through it.
+ *
+ * @example
+ * const linkedReader = createLinkReader({
```

Comment:
Where does `createLinkReader` come from? Maybe the import statement should be added here, right?

Reply:
Done 👍